source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,65 @@
+.. _ruby-get-started-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Edit your {+language+} application file
+
+      Navigate to your ``quickstart.rb`` file in the ``ruby-quickstart``
+      directory. Copy and paste the following code under the Bundler
+      code from the :ref:`ruby-quick-start-download-and-install` step 
+      of this tutorial to query the ``movies`` collection in the ``sample_mflix``
+      database:
+
+      .. literalinclude:: /includes/get-started/quickstart.rb
+         :language: ruby
+         :dedent:
+         :start-after: start-query
+         :end-before: end-query
+
+   .. step:: Assign the connection string
+
+      Replace the ``<connection string>`` placeholder with the 
+      connection string that you copied from the :ref:`ruby-get-started-connection-string`
+      step of this tutorial.
+
+   .. step:: Run your {+language+} application
+
+      In your project directory, run the following shell command to run the application:
```

Comment:
S: Be specific about where to run this from since the first step had them navigate away from the directory
```suggestion
      Navigate back to your ``ruby-quickstart`` directory and run the following shell command to run the application:
```